CONTRIBUTING.md

```diff
@@ -14,7 +14,7 @@ Pull requests are the best way to propose changes to the codebase (we use [Githu
 
 1. Fork the repo and create your branch from `master`.
 2. If you've added code that should be tested, add tests.
-3. If you've changed APIs, update the documentation and API Schema definitions.
+3. If you've changed APIs, update the documentation and API Schema definitions (see `tools/generate_grpc_docs.sh` and `tools/generate_openapi_models.sh`)
```

Comment:
You also link straight to the dev. documentation for API changes https://github.com/qdrant/qdrant/blob/master/docs/DEVELOPMENT.md#api-changes=